File: 10810.c
#include <stdio.h>
int main(){
    int n,m;
    int i,j,k;
    int ball[100]={0};
    scanf("%d %d",&n,&m);
    for (int l=0; l<m; l++){
        scanf("%d %d %d",&i,&j,&k);
        for (int idx=i; idx<=j; idx++) ball[idx-1]=k;
    }
    for (int idx=0; idx<n; idx++) printf("%d ",ball[idx]);
    printf("\n");
    return 0;
}

File: 10810.py
#Naive
input=open(0).readline
n,m=map(int,input().split())
ball=[0 for _ in range(n)]
for _ in range(m):
    i,j,k=map(int,input().split())
    for idx in range(i,j+1): ball[idx-1]=k
print(*ball)